packages/atomic/src/components/commerce/atomic-commerce-pager/atomic-commerce-pager.tsx

```diff
@@ -122,7 +122,7 @@ export class AtomicCommercePager
               this.focusOnFirstResultAndScrollToTop();
             }}
           />
-          <PagerPageButtons>
+          <PagerPageButtons label={this.bindings.i18n.t('pagination')}>
```

Comment:
Nitpick: 
Instead of doing it like this, modify `PagerPageButtonProps` to receive `i18n`, and do `i18n.t('pagination')` directly in the functional components.

Rationale is that it's easier to update the label everywhere at once if necessary, and it's not the type of value that needs to be unique per use case (ie: all use case should use the exact same label description)